docs/distributions/fedora/faq.md

```diff
@@ -5,3 +5,23 @@ If you already have Bootcamp installed, you might notice that the boot option fo
 # My boot hangs before getting to the installer
 
 This may be due to differences between USB-C to USB-A adapters. Try a different one if it is not working.
+
+# My touchbar is blank
+
+This is becuase of a firmware bug. Reboot into MacOS recovery (hold `CMD+R` while booting), then boot back into Fedora.
+
+# My touchbar is flickering
+
+Your touchbar is probably broken at the hardware level. Pay apple to fix it.
+
+# My keyboard won't light up (only applies to some hardware)
```

Comment:
Probably stress that this bug is only on Fedora on MacBooks with the Magic Keyboard

Reply:
The keyboard backlight works for most people now, but it isnt reliable. Even users with the same model have varied experiences.

Reply:
Ok then atleast stress that it's exclusive to Fedora then.